### ISSUE TITLE
Take hit Animation

### ISSUE THREAD

**Issue by M-Pages** [DarkFutureStudio/Deft-Rage#21]
### Animations for taking hits
Making animation for while the character is getting punched.

**Comment by ErfanTime** [comment]
is it not duplicate with #22 ?

**Comment by M-Pages** [comment]
Not exactly...because this animation will play when the player is getting hit but #22 is when the combo is finished and he get force
